azure-pipelines.yml

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2019 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+# Imports the shared Azure CI tools
+resources:
+  repositories:
+    - repository: templates
+      type: github
+      name: shotgunsoftware/tk-ci-tools
+      ref: refs/heads/run_tests
```

Comment:
Oh and of course this will need to switch over to master at some point as well, yeah?

Reply:
Yup. Once you've done the code review for it ;)
https://github.com/shotgunsoftware/tk-ci-tools/pull/2

Reply:
Yup. Once you've done the code review for it ;)
https://github.com/shotgunsoftware/tk-ci-tools/pull/2